Server/Admin/routers/admin_auth.py

```
@@ -693,23 +707,53 @@ def admin_password_reset_confirm(data: AdminPasswordResetConfirm, request: Reque
+        
+        if token:
+            from jose import jwt as jose_jwt
+            # Decode without verification to extract claims
+            payload = jose_jwt.decode(
+                token,
+                SECRET_KEY,
+                options={"verify_signature": False, "verify_exp": False, "verify_aud": False}
+            )
+            client_id = payload.get("aud") or payload.get("client_id") or "admin_web"
+            service = payload.get("service") or "admin"
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)